src/views/feedback/components/FeedBackDiag.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <el-dialog v-model="dialogTableVisible">
+    <span class="dialogTableVisible-title">反馈内容</span>
+    <el-card class="feedback-card">
+      <div class="feedback-image-preview">
+        <el-image
+          class="dialogCard-img"
+          :src="currentFeedBack?.imageUrl[0]"
+          :zoom-rate="1.2"
+          :max-scale="7"
+          :min-scale="0.2"
+          :preview-src-list="currentFeedBack?.imageUrl"
+          :initial-index="4"
+          fit="cover"
+          title="点击预览哦"
+        />
+      </div>
+      <div class="dialogCard-title">
+        <span>{{ currentFeedBack?.content }}</span>
+      </div>
+      <template #footer>
+        <el-input
+          v-model="feedBackContent"
+          :rows="7"
+          type="textarea"
+          placeholder="请输入回复内容"
+        />
+        <div class="dialogCard-btn">
+          <el-button @click="OnCancelMsg">取消</el-button>
+          <el-button type="primary" @click="OnHandleMsg">发送</el-button>
+        </div>
+      </template>
+    </el-card>
+  </el-dialog>
+</template>
+<script setup lang="ts">
+import { FeedBackInfo, ReplyInfo } from '@/types/feedback'
+import { ref, defineExpose } from 'vue'
+import { ElMessage } from 'element-plus'
+import { useFeedBackStore } from '@/store'
+const feedBackStore = useFeedBackStore()
+const { replyToUser, updateStateReply } = feedBackStore
+//打开反馈
+const dialogTableVisible = ref(false)
+
+//反馈内容
+const feedBackContent = ref('')
+
+// 接受参数
+const { currentFeedBack, replyMsg } = defineProps<{
+  currentFeedBack: FeedBackInfo | undefined
+  replyMsg: ReplyInfo
+}>()
+console.log(currentFeedBack)
+
+// 关闭对话框
+const closeDialog = () => {
+  dialogTableVisible.value = false
+  // 清空 textarea 内容
+  feedBackContent.value = ''
+}
+
+// 打开对话框
+const openDialog = () => {
+  dialogTableVisible.value = true
+}
+
+// 回复消息
+const OnHandleMsg = async () => {
+  if (feedBackContent.value === '') {
+    ElMessage.warning('回复内容不能为空')
```

Comment:
![image](https://github.com/Komorebi18/Item-Management/assets/82131679/b9bba2d5-8634-4d26-9b25-a7b12d9902aa)
```suggestion
    ElMessage.warning('回复内容不能为空');
    return;
```